cloud/misc/proxmox_kvm.py

```diff
@@ -0,0 +1,1055 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Abdoul Bah (@helldorado) <abdoul.bah at alterway.fr>
+
+"""
+Ansible module to manage Qemu(KVM) instance in Proxmox VE cluster.
+This module is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This software is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this software.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: proxmox_kvm
+short_description: Management of Qemu(KVM) Virtual Machines in Proxmox VE cluster.
+description:
+  - Allows you to create/delete/stop Qemu(KVM) Virtual Machines in Proxmox VE cluster.
+version_added: "2.3"
+author: "Abdoul Bah (@helldorado) <abdoul.bah at alterway.fr>"
+options:
+  acpi:
+    description:
+      - Specify if ACPI should be enables/disabled.
+    required: false
+    default: "yes"
+    choices: [ "yes", "no" ]
+    type: boolean
+  agent:
+    description:
+      - Specify if the QEMU GuestAgent should be enabled/disabled.
+    required: false
+    default: null
+    choices: [ "yes", "no" ]
+    type: boolean
+  args:
+    description:
+      - Pass arbitrary arguments to kvm.
+      - This option is for experts only!
+    default: "-serial unix:/var/run/qemu-server/VMID.serial,server,nowait"
+    required: false
+    type: string
+  api_host:
+    description:
+      - Specify the target host of the Proxmox VE cluster.
+    required: true
+  api_user:
+    description:
+      - Specify the user to authenticate with.
+    required: true
+  api_password:
+    description:
+      - Specify the password to authenticate with.
+      - You can use C(PROXMOX_PASSWORD) environment variable.
+    default: null
+    required: false
+  autostart:
+    description:
+      - Specify, if the VM should be automatically restarted after crash (currently ignored in PVE API).
+    required: false
+    default: "no"
+    choices: [ "yes", "no" ]
+    type: boolean
+  balloon:
+    description:
+      - Specify the amount of RAM for the VM in MB.
+      - Using zero disables the ballon driver.
+    required: false
+    default: 0
+    type: integer
+  bios:
+    description:
+      - Specify the BIOS implementation.
+    choices: ['seabios', 'ovmf']
+    required: false
+    default: null
+    type: string
+  boot:
+    description:
+      - Specify the boot order -> boot on floppy (a), hard disk (c), CD-ROM (d), or network (n).
+      - You can combine to set order.
+    required: false
+    default: cnd
+    type: string
+  bootdisk:
+    description:
+      - Specify if booting from specified disk is enabled.
```

Comment:
"Boot from the specified disk."

Can you list multiple disks for it to try in order? May be worth clarifying that point.